### DISCUSSION TITLE
우커머스 고유통관부호번호 넣는 방법

### DISCUSSION THREAD

**Discussion by LEEARA304** [woocommerce/woocommerce#52519] [Checkout Flow]
우커머스 고유통관부호번호 넣는 방법이 궁급합니다.

**Comment by nielslange** [comment]
Hello @LEEARA304 👋

Thanks for reaching out. I translated your message to English using Google Translate.

**Title:**
How to enter Woocommerce unique customs clearance code

**Description:**
I urgently need help entering the Woocommerce unique customs clearance code number.

Can you explain what you mean by "customs clearance"? Do you want to add an extra field to your checkout form where customers can enter such a number?

**Comment by LEEARA304** [comment]
Yes. I want to add an extra field to my checkout form.
-----Original Message-----

**Comment by nielslange** [comment]
> Yes. I want to add an extra field to my checkout form.

Thanks for clarifying your use case. You can use the [additional fields API](https://developer.woocommerce.com/docs/cart-and-checkout-additional-checkout-fields/) to add additional fields to the checkout form.
